src/pages/register.tsx

```diff
@@ -51,7 +51,6 @@ export default function Page() {
     }
   }
 
-  // TODO: Temp fix for hydration issues.
```

Comment:
can't we skeletonise this for hydration rather than return null

Reply:
Removed the return null on loading as the passing the updated isLoading variable into <Registration/> seemed to stop the hydration errors.